packages-ts/gauntlet-terra/src/commands/ledgerKey.ts

```diff
@@ -51,19 +53,23 @@ export class LedgerKey extends Key {
     const transport = await TransportNodeHid.create()
     const ledgerConnector = new LedgerTerraConnector(transport)
     const response = await ledgerConnector.initialize()
-    this.checkForErrors(response)
+    this.assertNoErrors(response)
 
     return {
       ledgerConnector,
       terminateConnection: transport.close.bind(transport),
     }
   }
 
-  private static pathStringToArray(path: string): Array<number> {
-    return path.split("'/").map((item) => parseInt(item))
+  private static bip44PathtoArray(path: string): Array<number> {
```

Comment:
This is a nice util function that we should move to `gauntlet-core` at one point.